NOTICE.md

```diff
@@ -0,0 +1,205 @@
+# Notices for Eclipse Metro
+
+This content is produced and maintained by the Eclipse Metro project.
+
+* Project home: https://projects.eclipse.org/projects/ee4j.metro
+
+## Trademarks
+
+Eclipse Metro is a trademark of the Eclipse Foundation.
+
+## Copyright
+
+All content is the property of the respective authors or their employers. For
+more information regarding authorship of content, please consult the listed
+source code repository logs.
+
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v. 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License v. 1.0
+which is available at http://www.eclipse.org/org/documents/edl-v10.php. This
+Source Code may also be made available under the following Secondary Licenses
+when the conditions for such availability set forth in the Eclipse Public
+License v. 2.0 are satisfied: GNU General Public License, version 2 with the GNU
+Classpath Exception which is available at
+https://www.gnu.org/software/classpath/license.html.
+
+SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause OR GPL-2.0 WITH
+Classpath-exception-2.0
+
```

Comment:
I know it's wrong. We should keep it as it is now. Will be fixed later when https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.metro is fixed.

Reply:
do you have a pointer to a bug asking for it?

Reply:
https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.metro is fixed now, updating NOTES to be OK.